src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -1227,6 +1227,8 @@ module Base = struct
                  }
              ; delta
              ; events = _ (* This is for the snapp to use, we don't need it. *)
+             ; call_data =
+                 _ (* This is for the snapp to use, we don't need it. *)
```

Comment:
Doesn't this have to be part of the input to the snark? How is it tied to the proof aka function call?